src/components/Tabber/Tabber.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+
+import { networkListMock } from './networkListMock';
+
+import TabsTabber from './components/Tabs/TabsTabber';
+
+describe('TabsTabber Component', () => {
+  it('renderiza todos os itens da lista de redes sociais', () => {
+    render(
+      <div>
+        {networkListMock.map((socialItem) => (
+          <TabsTabber
+            key={socialItem.id}
+            socialItem={socialItem}
+            selectedTab={socialItem.id}
+            onTabClick={() => {}}
+          />
+        ))}
+      </div>
+    );
+
+    networkListMock.forEach((socialItem) => {
+      const item = screen.getByText(socialItem.id);
+      expect(item).toBeInTheDocument();
+    });
+  });
```

Comment:
em vez de você colocar um `map` e um `foreach` voce pode usar o `it.each` pra fazer esses testes
https://dev.to/bgord/simplify-repetitive-jest-test-cases-with-test-each-310m
https://www.youtube.com/watch?v=0dRPPmRYM3o&t=1939s minuto: 7:10